ext/objspace/lib/objspace.rb

```diff
@@ -0,0 +1,87 @@
+# frozen_string_literal: true
+
+require 'objspace.so'
+
+module ObjectSpace
+  class << self
+    private :_dump
+    private :_dump_all
```

Comment:
I wish I could use the `Primitive.` system rather than call private methods, but I'm not sure wether it can be used in `ext/`.

Reply:
I agree, but this seems fine for now.